schema.sql

```diff
@@ -1,5 +1,15 @@
 /* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
+postgres=# \c vet_clinic
+CREATE TABLE animals(
+    id serial PRIMARY KEY,
```

Comment:
- [ ] [OPTIONAL] Kindly make the id column not null and autoincremented so you don't have to insert it manually like:
```
id int NOT NULL IDENTITY(1,1)
```